web/modules/custom/dpl_event/src/PriceFormatter.php

```diff
@@ -35,10 +70,7 @@ public function formatPrice(string $price_string): string {
       // Format the numeric part of the price using formatRawPrice.
       $formatted_price = $this->formatRawPrice($price_string);
 
-      // Add the 'kr.' suffix for now.
-      // For multi-currency support this should be replaced by a configurable
-      // suffix and appropriate separators.
-      return $formatted_price . ' kr.';
+      return "{$this->currencyPrefix}$formatted_price{$this->currencySuffix}";
```

Comment:
This is a matter of taste. But if values are being concatenated like this I prefer to work with arrays and the in the end use implode.
Like:
```php
$price[] = $this->currencyPrefix;
$price[] = $this->formatRawPrice($price_string);
$price[] = $this-> currencySuffix;
return implode($price);
```
or:
```php
return implode([
  $this->currencyPrefix;
  $this->formatRawPrice($price_string);
  $this-> currencySuffix
]);
```

Reply:
I respect your preference, but I prefer my method - i feel like it's more logical and readable :)